modules/perception/obstacle/camera/lane_post_process/common/type.h

```diff
@@ -24,47 +24,25 @@
 #include "Eigen/Core"
 #include "opencv2/opencv.hpp"
 
-#include "modules/perception/proto/perception_obstacle.pb.h"
 #include "modules/common/log.h"
 #include "modules/perception/obstacle/camera/lane_post_process/common/base_type.h"
+#include "modules/perception/proto/perception_obstacle.pb.h"
 
 #ifndef MODULES_PERCEPTION_OBSTACLE_CAMERA_LANE_POST_PROCESS_COMMON_TYPE_H_
 #define MODULES_PERCEPTION_OBSTACLE_CAMERA_LANE_POST_PROCESS_COMMON_TYPE_H_
 
 namespace apollo {
 namespace perception {
 
-#ifndef MAX_LANE_HISTORY
-#define MAX_LANE_HISTORY 10
```

Comment:
please convert to gflags or use proto conf. That will be more equivalent to the original code

Reply:
reversed changes on type.h. We might need to re-name some of the params in gflags as the current naming could be conflict. Might do the change in some future versions.